CHANGELOG.md

```diff
@@ -68,3 +68,86 @@
 [#21]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/21
 [#22]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/22
 [#23]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/23
+
+# v2.0.0 (2024-06-14)
+
+## Kit Features
+
+* Move code to core kit ([#1])
+* Add libbpf ([#24])- Thanks ndbaker1!
+* Use Bottlerocket Settings SDK for settings models - ([#7])
+
+## OS Changes
+
+* Add libz dependency ([#31])
+* Update Rust dependencies ([#34])
+* Add Neuron ([#21])
+* Amazon SSM Agent dynamically linked binaries ([#22])
```

Comment:
#8 was the last PR merged for 2.0.0 per the [commit history](https://github.com/bottlerocket-os/bottlerocket-core-kit/commits/6939dd235ab4274e633b0022ffc812e00334648d/) - anything after that was a different release.